__tests__/integration.js

```diff
@@ -4,6 +4,7 @@ const rmfr = require('rmfr')
 const nextBuild = require('next/dist/build').default
 const nextExport = require('next/dist/export').default
```

Comment:
I think we can drop these two lines now right?

Reply:
Oh, yep!

Reply:
Actually, no, these are still used by `compileNextjsWithMockFunction`. That's still used by the `options.onContent` test.